<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectPage</class>
 <widget class="QDialog" name="ConnectPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>USB</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>设备PID</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="USBServerPID">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>设备VID</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="USBServerVID">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="USBServerScanButton">
           <property name="text">
            <string>扫描设备</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>总线地址</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="USBDeviceBusNumberComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>设备地址</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="USBDeviceAddressComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="USBServerConnectButton">
           <property name="text">
            <string>连接设备</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="USBServerDisconnectButton">
           <property name="text">
            <string>断开连接</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TCPServerScanTab">
      <attribute name="title">
       <string>TCP服务器</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="TCPServerScanHorizontalLayout">
         <item>
          <layout class="QHBoxLayout" name="TCPServerAddressHorizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>服务器IP地址</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="TCPServerAddress">
             <property name="text">
              <string>192.168.1.100</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="TCPServerPortHorizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>服务器端口号</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="TCPServerPort">
             <property name="text">
              <string>8899</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="TCPServerScanButton">
           <property name="text">
            <string>扫描设备</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="TCPSocketConnecHorizontalLayout">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>最大连接个数</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="TCPSocketConnectCount"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>设备IP地址</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="TCPServerSocketAddress">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="TCPSocketConnectButton">
           <property name="text">
            <string>连接设备</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="TCPSocketDisconnectButton">
           <property name="text">
            <string>断开连接</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
